src/features/page_view_event/aggregate/index.js

```diff
@@ -102,6 +103,10 @@ export class Aggregate extends AggregateBase {
     queryParameters.fp = firstPaint.current.value
     queryParameters.fcp = firstContentfulPaint.current.value
 
+    if (this.timeKeeper?.ready) {
+      queryParameters.timestamp = this.timeKeeper.convertRelativeTimestamp(now())
```

Comment:
Is this timestamp intended to line up with the page load time or simply just match the submission time?

Reply:
The CDD stats it is the submission time.